raiden/ui/cli.py

```diff
@@ -238,8 +271,31 @@ def run(ctx, external_listen_address, **kwargs):
 
     app_.raiden.register_registry(app_.raiden.chain.default_registry)
 
-    console = Console(app_)
-    console.start()
+    if ctx.params['rpc']:
+        # instance of the raiden-api
+        raiden_api = RaidenAPI(app_.raiden)
+        # wrap the raiden-api with rest-logic and encoding
+        rest_api = RestAPI(raiden_api)
+        # create the server and link the api-endpoints with flask / flask-restful middleware
+        api_server = APIServer(rest_api)
+        # run the server
+        Greenlet.spawn(api_server.run, ctx.params['port'], debug=True, use_evalex=False)
```

Comment:
Change `debug=False` and your problem with having the `app.run()` function being executed twice will go away.